technic/machines/MV/hydro_turbine.lua

```diff
@@ -75,9 +75,11 @@ minetest.register_node("technic:hydro_turbine", {
 	},
 	paramtype2 = "facedir",
 	groups = {snappy=2, choppy=2, oddly_breakable_by_hand=2,
-	technic_machine=1, technic_mv=1},
+		  technic_machine=1, technic_mv=1, axey=2, handy=1},
```

Comment:
Minor indent / style
```suggestion
		technic_machine=1, technic_mv=1, axey=2, handy=1},
```